apis/identity/v1alpha1/referencers.go

```diff
@@ -83,12 +82,12 @@ func (mg *IAMGroupUserMembership) ResolveReferences(ctx context.Context, c clien
 	if err != nil {
 		return err
 	}
-	mg.Spec.ForProvider.UserName = &user.ResolvedValue
+	mg.Spec.ForProvider.UserName = user.ResolvedValue
 	mg.Spec.ForProvider.UserNameRef = user.ResolvedReference
 
 	// Resolve spec.forProvider.groupName
 	group, err := r.Resolve(ctx, reference.ResolutionRequest{
-		CurrentValue: *mg.Spec.ForProvider.GroupName,
```

Comment:
For reviewers: this (and same situation below) is the primary need for this fix. I initially thought to just use `reference.FromPtrValue` but then realized these fields should not be optional anyway.